GitVersion.yml

```diff
@@ -1,4 +1,7 @@
 mode: ContinuousDeployment
+branches:
+  release:
+    is-release-branch: false
```

Comment:
This is the crux of the PR which stops GitVersion using `release/*` branches as part of calculating versions for builds.